components/Buttons/src/MDCFloatingButton.m

```diff
@@ -105,15 +105,13 @@ - (CGSize)intrinsicContentSize {
   }
 }
 
-#pragma mark - Subclassing
-
-- (UIBezierPath *)boundingPath {
-  return [UIBezierPath bezierPathWithOvalInRect:self.bounds];
+- (void)layoutSubviews {
+  // We have to set cornerRadius before laying out subviews so that the boundingPath is correct.
+  self.layer.cornerRadius = CGRectGetHeight(self.bounds) / 2;
```

Comment:
I thought the whole point was to let clients set the corner radius and that we would not erase their value. What about subclasses of MDCFloatingButton?

Reply:
I spoke with @samnm on Friday. We determined that `cornerRadius` should not be problematic for internal clients as originally authored by @samnm and reproduced in this PR. The previous approach (using a new property) should not be necessary as an intermediate step. 

Simply removing the behavior (but allowing subclasses to override for now) is a cleaner, direct step in the correct direction.

Reply:
sgtm

Reply:
@romoore sorry to revive an old PR, but any reason why here it is `CGRectGetHeight` whereas it is replacing `CGRectGetWidth`